rs/registry/canister/src/common/test_helpers.rs

```diff
@@ -196,6 +197,7 @@ pub fn registry_add_node_operator_for_node(
     {
         let node_operator_record = NodeOperatorRecord {
             node_allowance,
+            rewardable_nodes: btreemap! { "type0".to_string() => 0, "type1".to_string() => 28 },
```

Comment:
Just for my understanding, this is back-filling some data that happens to be correct now, but eventually we should be able to drop this, line, right? 

In other words, why is the constant 28 used here?

Reply:
No, that is a test helper. It's only used in tests.